test/asset_host_test.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "test_helper"
+
+describe Plek do
+  describe "retreiving the public_asset_host" do
+    it "should return the GOVUK_ASSET_HOST env variable" do
```

Comment:
Can we use "returns" and drop the "should" so the spec docs are more definitive? Here and elsewhere in this file.